src/shared-components/dropdown/style.js

```diff
@@ -9,7 +9,7 @@ import {
   TYPOGRAPHY_CONSTANTS,
 } from '../../constants';
 
-const optionsContainerMaxHeight = '250px';
+const optionsContainerMaxHeight = '206px';
```

Comment:
I think we should be cautious changing the behavior of other Dropdown usages:

250px max-height:

<img width="541" alt="Screen Shot 2020-07-02 at 1 09 08 PM" src="https://user-images.githubusercontent.com/13544620/86395430-da651d80-bc65-11ea-80f0-8313367cad01.png">

208px max-height:

<img width="543" alt="Screen Shot 2020-07-02 at 1 09 15 PM" src="https://user-images.githubusercontent.com/13544620/86395448-e18c2b80-bc65-11ea-9529-183417d21d07.png">

So I think we should try and scope the change to fixing the particular case we're trying to fix. I don't know if any of the changes might require design resources. Some ideas (list is not: in any particular order, prescriptive, nor exhaustive):

1. Making max-height adjustable via props to the `radiance-ui` component.
1. Scoping some `emotion` css usage in PocketDerm to target the particular `ul` instance we're trying to fix. 
1. Just making the height of the parent container a bit larger to accommodate the 250px max-height. 
1. Allowing the particular Dropdown instance we're concerned with to be positioned `relative` instead of `absolute` so the container height expands and shrinks as the Select expands and shrinks.

Reply:
Ah nice. I like the idea of making `max-height` adjustable via props. I will refactor 👍